.github/workflows/builder_installation_assistant.yml

```diff
@@ -12,22 +12,35 @@ on:
         description: "Is stage?"
         type: boolean
         default: false
+      checksum:
+        description: "Add checksum"
+        type: boolean
+        default: false
       id:
         description: "ID used to identify the workflow uniquely."
         type: string
         required: false
   workflow_call:
     inputs:
+      WAZUH_INSTALLATION_ASSISTANT_REFERENCE:
```

Comment:
Use lowercase for the input names: `WAZUH_INSTALLATION_ASSISTANT_REFERENCE`
Lines: 7 and 25, and update the references.